bitnami/kubeapps/templates/kubeappsapis/serviceaccount.yaml

```diff
@@ -1,13 +1,22 @@
+{{- if .Values.kubeappsapis.serviceAccount.create }}
```

Comment:
I think service account is mandatory if `packaging.flux.enabled` and `rbac.create` are enabled

Reply:
Same as in the previous comment, `serviceAccount.create` is `true` by default. But users can specify custom account name, or use an existing account name by setting `serviceAccount.create=false`.